audbcards/core/datacard.py

```diff
@@ -71,11 +71,14 @@ def example(self) -> typing.Optional[str]:
         min_dur = 0.5
         max_dur = 300  # 5 min
         durations = self._dataset.file_durations
-        selected_duration = np.median(
-            [d for d in durations if d >= min_dur and d <= max_dur]
-        )
-        if np.isnan(selected_duration):
+        selected_durations = [
+            d for d in durations
+            if d >= min_dur and d <= max_dur
+        ]
+        if len(selected_durations) == 0:
             return None
+        selected_duration = np.median(selected_durations)
```

Comment:
The only thing that I am unsure about is the reason why calling the pandas median function on an empty list has to be avoided.  Stil I would say that this can be merged.

Reply:
I guess you are referring to the two lines before:
```python
if len(selected_durations) == 0:
    return None
```

The goal of `Datacard.example` is to select an example file. If the list of durations is empty we do not need to continue.

Reply:
But, why exactly I changed it here, I also do not remember. I guess I find it easier to understand this way ;)